container/devel/Dockerfile.openqa.leap

```
@@ -0,0 +1,34 @@
+FROM registry.opensuse.org/opensuse/leap:15.6
+
+RUN zypper ar -G https://download.opensuse.org/repositories/devel:/openQA/15.6 devel-openqa \
+        && zypper ar -G https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.6/15.6/ devel-openqa-leap \
+        && zypper ar -G https://download.opensuse.org/repositories/SUSE:/CA/15.6/SUSE:CA.repo \
+        && zypper -n ref \
+    && true
+
+RUN zypper -n install ca-certificates-suse
+
+# openqa
+RUN zypper -n ref && zypper -n install \
+        os-autoinst-devel openQA-devel \
+        expect \
+        nodejs \
+        openssh-clients \
+    && true
+
+# scripts
+RUN zypper -n install \
+        os-autoinst-scripts-deps \
+        os-autoinst-scripts-deps-devel \
+        python3-pytest \
+        python3-virtualenv \
+    && true
```

How about

```suggestion
RUN zypper -n ref && zypper -n install \
        os-autoinst-devel openQA-devel \
        expect \
        nodejs \
        openssh-clients \
        ca-certificates-suse \
        os-autoinst-scripts-deps \
        os-autoinst-scripts-deps-devel \
        python3-pytest \
        python3-virtualenv \
    && zypper clean
```

and I think this shows a flaw in https://github.com/os-autoinst/scripts/blob/master/dist/rpm/os-autoinst-scripts-deps.spec#L32 compared to https://github.com/os-autoinst/os-autoinst/blob/master/dist/rpm/os-autoinst.spec#L122 that os-autoinst-scripts development requirements should include main+test requires. Maybe better fix that there? And the other dependencies: When they are actually needed for development then they should be part of os-autoinst-devel or openQA-devel